src/components/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+
+export type Props = {
+  comments: Comment[];
+};
+
+export const CommentsList: React.FC<Props> = ({ comments }) => (
+  <div className="section__comments">
+    {comments.map(element => (
+      <div
+        key={element.id}
+        className="comment-card"
+      >
+        <CommentInfo comments={element} />
```

Comment:
```suggestion
        <CommentInfo comment={element} />
```
Зверни увагу на назву. Ти сюди передаєш один комент, а не декілька